app/src/main/res/layout/license_row.xml

```diff
@@ -1,44 +1,45 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:foreground="?attr/selectableItemBackground"
```

Comment:
`Attribute android:foreground has no effect on API levels lower than 23 (current min is 21) `, but with `android:background` it works fine and doesn't show the warning.

Reply:
Have you tested it in an actual device? That was required for the ripple effect. Please test your changes and lmk

Reply:
I've tested all the changes which I pushed. Wrote images: raw and dmg. Checked the styles, how it looks after all. 
All works fine. 
You don't trust to commits but trust to self-made screenshots of UI?

Reply:
No, I'll run the app to see if it works as intended, I just wanted to know if you checked.